python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,657 @@
+    "_020E": meta(Sex.MALE, 45, 54, HealthInsurancePopulation.WITHOUT),
+    "_021E": meta(Sex.MALE, 55, 64),
+    "_022E": meta(Sex.MALE, 55, 64, HealthInsurancePopulation.WITH),
+    "_023E": meta(Sex.MALE, 55, 64, HealthInsurancePopulation.WITHOUT),
+    "_024E": meta(Sex.MALE, 65, 74),
+    "_025E": meta(Sex.MALE, 65, 74, HealthInsurancePopulation.WITH),
+    "_026E": meta(Sex.MALE, 65, 74, HealthInsurancePopulation.WITHOUT),
+    "_027E": meta(Sex.MALE, 65, None),
+    "_028E": meta(Sex.MALE, 65, None, HealthInsurancePopulation.WITH),
+    "_029E": meta(Sex.MALE, 65, None, HealthInsurancePopulation.WITHOUT),
+}
+
+# Health insurance by female suffixes.
+HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES = {
+    "_031E": meta(Sex.FEMALE, 0, 6),
+    "_032E": meta(Sex.FEMALE, 0, 6, HealthInsurancePopulation.WITH),
+    "_033E": meta(Sex.FEMALE, 0, 6, HealthInsurancePopulation.WITHOUT),
+    "_034E": meta(Sex.FEMALE, 6, 18),
+    "_035E": meta(Sex.FEMALE, 6, 18, HealthInsurancePopulation.WITH),
+    "_036E": meta(Sex.FEMALE, 6, 18, HealthInsurancePopulation.WITHOUT),
+    "_037E": meta(Sex.FEMALE, 19, 25),
+    "_038E": meta(Sex.FEMALE, 19, 25, HealthInsurancePopulation.WITH),
+    "_039E": meta(Sex.FEMALE, 19, 25, HealthInsurancePopulation.WITHOUT),
+    "_040E": meta(Sex.FEMALE, 26, 34),
+    "_041E": meta(Sex.FEMALE, 26, 34, HealthInsurancePopulation.WITH),
+    "_042E": meta(Sex.FEMALE, 26, 34, HealthInsurancePopulation.WITHOUT),
+    "_043E": meta(Sex.FEMALE, 35, 44),
+    "_044E": meta(Sex.FEMALE, 35, 44, HealthInsurancePopulation.WITH),
+    "_045E": meta(Sex.FEMALE, 35, 44, HealthInsurancePopulation.WITHOUT),
+    "_046E": meta(Sex.FEMALE, 45, 54),
+    "_047E": meta(Sex.FEMALE, 45, 54, HealthInsurancePopulation.WITH),
+    "_048E": meta(Sex.FEMALE, 45, 54, HealthInsurancePopulation.WITHOUT),
+    "_049E": meta(Sex.FEMALE, 55, 64),
+    "_050E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITH),
+    "_051E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITHOUT),
+    "_052E": meta(Sex.FEMALE, 65, 74),
+    "_053E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITH),
+    "_054E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITHOUT),
+    "_055E": meta(Sex.FEMALE, 65, None),
+    "_056E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITH),
+    "_057E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITHOUT),
+}
+
+# Given the Prefix: eg.
+# {"C27001A":{MetadataKey.RACE: Race.WHITE_ALONE}}
+# will combine with the provided suffixes eg.
+# {
+#   "_002E":meta(None,0, 19),
+#   "_003E":meta(None,0, 19,HealthInsurancePopulation.WITH)
+#   ...
+# }
+# to create the ACS variable params.
+# ?for=state&get=C27001A_002E,C27001A_003E...
+
+
+def format_params(prefixes, suffixes, isCounty=False):
+    groups = []
+    for prefix in prefixes:
+        for suffix in suffixes:
+            groups.append(prefix+suffix)
+    vars = ','.join(groups)
+
+    return {'for': 'county' if isCounty else 'state', "get": vars}
```

Comment:
another nit: it would be nice to either use double quotes everywhere or single quotes everywhere in the file